wiki/Gameplay/Input_device/Graphic_tablet/en.md

```diff
@@ -0,0 +1,10 @@
+# Graphic Tablet
+
+A **graphic tablet** is an input device mainly used for drawing but can be used in osu! as an input device. 
+A tablet can be used instead of the mouse to aim. This is a very common play style in osu!
+
+If you are using a tablet PC or your tablet clicks are not handled correctly, you can enable the "OS TabletPC support" setting in the options menu.
```

Comment:
```suggestion
Notice to tablet players: The use of a Tablet PC or tablet clicks in osu! may sometimes be handled incorrectly To fix this, enable the `OS Tablet PC support` setting in the Options menu.
```
Not sure if my formatting is 100% correct on this, but the main thing is just fixing the POV